how-to/deploy/orchestrated/tiup.md

```diff
@@ -331,31 +331,31 @@ cat topology.yaml
 ```
 
 ```yaml
-# # Global variables are applied to all deployments and as the default value of
-# # them if the specific deployment value missing.
+# # Global variables are applied to all deployments and used as the default value of
+# # The deployments if a specific deployment value is missing.
```

Comment:
```suggestion
# # the deployments if a specific deployment value is missing.
```
Please see my commits and comment in https://github.com/pingcap/docs-cn/pull/2846 and align them in this PR. Thanks!

Reply:
@yikeke The file is updated. PTAL